pkg/scheduler/framework/runtime/framework.go

```diff
@@ -32,6 +32,7 @@ import (
 	"k8s.io/kube-scheduler/config/v1beta1"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config/scheme"
+	"k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
```

Comment:
Hi @ahg-g 

I should have mentioned this before, but reason why i left it as v1alpha1 is that there is a struct in this[file](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/v1alpha1/framework.go#L64) named `framework`.  I have preemptively named it `fwk` to avoid name conflcit pending your opinion.

But just one note, many packages outside package `framework` will import it without the alias like this:
`"k8s.io/kubernetes/pkg/scheduler/framework/runtime"`

Should I add an alias to these as well, to use `fwkruntime`?